app/client/rpc/routing.go

```diff
@@ -0,0 +1,54 @@
+package rpc
+
+import (
+	"github.com/julienschmidt/httprouter"
+)
+
+const (
+	HealthRoute            RouteKey = "Health"
+	VersionRoute           RouteKey = "Version"
+	BroadcastTxSyncRoute   RouteKey = "BroadcastTxSync"
+	BroadcastTxAsyncRoute  RouteKey = "BroadcastTxAsync"
+	BroadcastTxCommitRoute RouteKey = "BroadcastTxCommit"
+	QueryTxRoute           RouteKey = "QueryTx"
+)
+
+var RoutesMap = map[RouteKey]Route{
```

Comment:
Have you had a chance to look into whether there's a way to generate this from an OpenAPI spec definition?

cc @okdas for context

Reply:
Looking into this: https://github.com/deepmap/oapi-codegen and using [echo](https://github.com/labstack/echo) as HTTP server. 

I already have a working version locally. Next push will include the change for review